daemon/network.go

```diff
@@ -303,6 +303,10 @@ func (daemon *Daemon) createNetwork(cfg *config.Config, create types.NetworkCrea
 		return nil, errdefs.Forbidden(errors.New(`This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again.`))
 	}
 
+	if network.HasIPv6Subnets(create.IPAM) && !create.EnableIPv6 {
```

Comment:
I guess the `&& !create.EnableIPv6` is a bit redundant, but definitely not a blocker 😂

Reply:
Removed that bit.